sql-plugin/src/main/spark330/scala/com/nvidia/spark/rapids/shims/GpuIntervalUtils.scala

```diff
@@ -165,7 +165,11 @@ object GpuIntervalUtils {
     withResource(castStringToDTInterval(cv, t)) { ret =>
       if(ret.getNullCount > cv.getNullCount) {
         throw new IllegalArgumentException("Cast string to day time interval failed, " +
-            "may be the format is invalid, range check failed or overflow")
+            "may be the format is invalid, range check failed or overflow. " + 
+            "Do not support short mode, take DAY TO SECOND for example: " +  
+            "not support short mode: 100 10:30:40.999999, " +  
+            "only support the normal mode: "  +
+            "INTERVAL '100 10:30:40.999999' DAY TO SECOND")
```

Comment:
where does the term "short/normal mode" come from? Is there some doc we can reference in this message and use its terminology?

Reply:
ref : https://nvidia.github.io/spark-rapids/docs/compatibility.html#csv-ansi-day-time-interval

Currently, the RAPIDS Accelerator only supports the ANSI style.